docs/UserGuide.md

```diff
@@ -68,6 +68,51 @@ not be so obvious!
 1. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+#### 3.2.2. General Symbols and Syntax 
+
+The following table explains general symbols and syntax used in this User Guide.
+
+| Syntax |  What it means |
+|----------|-------------|
+| `command` |  The grey block specifies an execuable command which can be entered in the command box |
+| _italics_ | Italicised text specifies the terms related to tCheck's GUI |
+|<div markdown="block" class="alert alert-info"> :information_source: </div>  | An exclamation mark indicates that the following text is a note |
+|<div markdown="span" class="alert alert-primary">:bulb:  | An bulb indicates that the following text is a tip |
+
+#### 3.2.3. Command Syntax and Usage
+
+The following table explains technical terms used in this user guide.
+
+| Technical Term | What it means |
+| ---------------| --------------|
+| Command Word | The first word of a command. tCheck uses this command word to decide what king of actions it should perform |
+| Prefix |It is the input that follows behind a prefix, used to distinguish different kinds of parameters |
+| Parameter | Values given for specific operations. |
```

Comment:
I think you should describe it in more layman terms first. eg. "It is the input that follows behind a prefix". Then, give the explanation, that you have written here.

Reply:
Thank you! I am changing it.